Разметка всех страниц

arteman8111 opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2051453/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2051453-sedona-34/2/

arteman8111 committed:
Первые шаги

arteman8111 committed:
Update index.html

arteman8111 committed:
Update index.html

arteman8111 committed:
Почти сделанное ДЗ

arteman8111 committed:
Готовое ДЗ

baileys-li commented on index.html:
```diff
@@ -3,10 +3,116 @@
+    <a class="link-HtmlAcademy" href="https://htmlacademy.ru/study"><span class="visually-hidden">Сслыка на академию</span><img class="icon" src="" alt="Логотип Академии" ></a>
+  </footer>
   </body>
+</head>
```
Потерялся закрывающий тэг 😔

baileys-li replied:
Потерялся закрывающий тэг 😔

baileys-li commented on index.html:
```diff
@@ -3,10 +3,116 @@
+  <nav class="navigation">
+    <img class="logo" src="" alt="Логотип Седоны">
+    <ul class="navigation-list">
+      <li class="navigation-item"><a class="navigation-link navigation-link-current" href="index.html"><span>Главная</span></a></li>
```
А зачем обрачивать ещё в спан?

baileys-li commented on index.html:
```diff
@@ -3,10 +3,116 @@
+    <ul class="navigation-list navigation-user">
+      <li class="navigation-item"><a class="navigation-link" href="#"><span class="visually-hidden">Поиск</span></a></li>
+      <li class="navigation-item"><a class="navigation-link" href="#"><span class="visually-hidden">Поставить лайк</span></a></li>
+      <li class="navigation-item"><a class="navigation-link" href="#"><span class="visually-hidden">Кликбейт</span></a></li>
```
Там же это кнопка «Хочу сюда»

baileys-li commented on index.html:
```diff
@@ -3,10 +3,116 @@
+     </ul>
+    </section>
+    <address class="contacts-number">
+      <a class="contacts-number-link" href="tel:+7 (812) 812-12-12">+7 (812) 812-12-12</a>
```
Чтобы не было ошибки валидатора
```suggestion
      <a class="contacts-number-link" href="tel:+78128121212">+7 (812) 812-12-12</a>
```

baileys-li commented on index.html:
```diff
@@ -3,10 +3,116 @@
+    <h3 class="subscribe-title">Подпишитесь на рассылку</h3>
+    <p class="subscribe-description">Только полезная информация и никакого спама, честное бойскаутское!</p>
+    <form class="subscribe-form" action="https://echo.htmlacademy.ru/" method="post">
+    <label class="label-form" for="email"></label>
```
Пустой лейбл

baileys-li reviewed (changes requested):
Записал скринкаст в телегу

arteman8111 committed:
Update index.html

arteman8111 committed:
Переделал ДЗ. index.html и catalog.html

Пока что не доразобрал поле range, у меня там не связываются два инпута
Ну и также жду демку со звездами, хз что там
